gtda/sampling/sample_point_clouds.py

```diff
@@ -0,0 +1,177 @@
+"""Point-cloud–related plotting functions and classes."""
+# License: GNU AGPLv3
+
+import numpy as np
+from scipy.optimize import root_scalar
+
+from gtda.utils import validate_params
+from gtda.utils.intervals import Interval
+
+
+def sphere_sampling(n=1000, r=1, noise=0):
+    """Uniformly and randomly samples points from a round
+    sphere centered at the origin of 3-space.
```

Comment:
from a round sphere? There exists a sphere which is not round? Or are there different types of spheres? 

delete round here.